README.md

```diff
@@ -1,13 +1,20 @@
-# whitehall
-
-### Running the Application
-
-**Use GOV.UK Docker to run any commands that follow.**
+# Whitehall
 
 Whitehall is deployed in two modes:
 
 - 'admin' for publishers to create and manage content (e.g. <http://whitehall-admin.dev.gov.uk/government/admin/news/new>)
-- 'frontend' for rendering content (eg: <http://www.gov.uk/government/get-involved>, <https://www.gov.uk/world>)
+- 'frontend' for rendering some legacy content (see live examples that follow) (eg: <https://www.gov.uk/government/get-involved>, <https://www.gov.uk/world>)
```

Comment:
I think removing the links here, since there's a whole section below makes more sense.